lib/prmd/templates/schemata/link.md.erb

```diff
@@ -3,7 +3,7 @@
   response_example = link['response_example']
   link_schema_properties_template = Prmd::Template.load_template('link_schema_properties.md.erb', options[:template])
 -%>
-### <%= title %> <%= link['title'] %>
+### <a name="resource-<%= resource %>/<%= link['method'] %><%= link['href'] %>"><%= title %> <%= link['title'] %></a>
```

Comment:
Ah right, href should help as you suggest. So maybe just `link-#{method}-#{resource}-#{href}`?